src/middlewared/middlewared/plugins/interface/lag.py

```diff
@@ -11,30 +11,75 @@ class Config:
     @private
     def lag_setup(self, lagg, members, disable_capabilities, parent_interfaces, sync_interface_opts):
         name = lagg['lagg_interface']['int_interface']
-        self.logger.info('Setting up {}'.format(name))
+        self.logger.info(f'Setting up {name}')
 
         try:
             iface = netif.get_interface(name)
         except KeyError:
             iface = None
         else:
-            first_port = next(iter(iface.ports))
+            first_port = next(iter(iface.ports), None)
             if first_port is None or first_port[0] != members[0]['lagg_physnic']:
-                self.logger.info('Destroying existing %s as its first port has changed', name)
+                self.logger.info(f'Destroying {name} because its first port has changed')
```

Comment:
For logging, we intentionally don't use f-strings as William/Vlad have pointed out earlier that if we don't use f-strings, the logging messages depending on the logging facility can be used to aggregate messages